sdcm/remote.py

```diff
@@ -59,10 +59,10 @@ def __init__(self, host, user=None, port=None, config=None, gateway=None, forwar
         super(ConnectionCmdTimeout, self).__init__(host, user, port, config, gateway, forward_agent, connect_timeout, connect_kwargs)
         self.cmd_timeout = None
 
+    @opens
```

Comment:
you don't need it here, it inside the `super(ConnectionCmdTimeout, self).run(command, **kwargs)`